tools/harness-thci/OpenThread.py

```diff
@@ -1312,9 +1318,11 @@ def setPollingRate(self, iPollingRate):
             False: fail to set the data polling rate for sleepy end device
         """
         print '%s call setPollingRate' % self.port
-        print iPollingRate
+        # convert s to ms
+        iPollingRate *= 1000
+        print int(iPollingRate)
```

Comment:
use `print` style compatible with python3?

Reply:
@bukepo Currently THCI uses python 2.7 and there are a lot of python2-style statements in THCI files. If needed, we should change them all in the future.